rate/fetchers/currency_beacon.go

```diff
@@ -0,0 +1,149 @@
+package fetchers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"slices"
+	"time"
+
+	"github.com/Hukyl/genesis-kma-school-entry/rate"
+	"github.com/ericchiang/css"
+	"golang.org/x/net/html"
+)
+
+const (
```

Comment:
```suggestion
const (
	cssSelector            = "tbody tr > td:first-child > a"
	supportedCurrenciesURL = "https://currencybeacon.com/supported-currencies"
	baseURL                = "https://api.currencybeacon.com/v1/latest?api_key=%s&base=%s&symbols=%s"
)
```

this way you can remove // nolint:lll line :)